core/unit_test/Makefile

```diff
@@ -80,18 +80,18 @@ ifeq ($(KOKKOS_INTERNAL_USE_ROCM), 1)
         OBJ_ROCM = UnitTestMainInit.o gtest-all.o
         OBJ_ROCM += TestROCm_Init.o
         OBJ_ROCM += TestROCm_Complex.o
-#        OBJ_ROCM += TestROCm_RangePolicy.o
-# rocm.range_scan locking up
+        OBJ_ROCM += TestROCm_RangePolicy.o
         OBJ_ROCM += TestROCm_AtomicOperations.o
         OBJ_ROCM += TestROCm_Atomics.o
 # complex failing
         OBJ_ROCM += TestROCm_AtomicViews.o
         OBJ_ROCM += TestROCm_Other.o
+#        OBJ_ROCM += TestROCm_MDRange.o
```

Comment:
Is this intended as being commented out?

Reply:
This is a placeholder, the test is written, but it doen't currently compile.  With this disabled, by current overnight tests build and run to completion.  Getting this working is my critical path item.